subprojects/libero-core/src/main/groovy/com/connexta/libero/Libero.groovy

```diff
@@ -186,15 +237,16 @@ class Libero {
         git.checkout(branch: config.releaseName)
         mavenResult = maven.execute(buildRequest)
         if (mavenResult.getExitCode() != 0) {
+            log.error("Release Build Failed due to: ${mavenResult.executionException}")
             throw new IllegalStateException( "Release Build failed!!" );
         }
-        // TODO: dry run should either remove tags and reset, or provide advice on how to do so
         if (!options.dryRun) {
```

Comment:
I'm thinking that might be a separate ticket :-)
At least the testing part, since there are several similar things that I'd ideally like to have libero test before attempting anything. I'll look into where I could stick the warning really quick though.